community/libplist/hack-autotools.patch

```diff
@@ -0,0 +1,25 @@
+For _some_ reason autotools doesn't want to run this check, even if run with --with-cython,
+so let's just remove it...
```

Comment:
Did you try `--with-cython=cython2` before patching?

Reply:
Ah, already did that but forgot to remove the files. Thanks